### PR TITLE
metrics: build and push to docker job

### PR DESCRIPTION
**Pull request by wozniakjan**
https://ci.openshift.redhat.com/jenkins/job/push_origin_metrics_release

cc: @jcantrill

### CODE REVIEW

File: sjb/config/test_cases/push_origin_metrics_release.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      chmod a+rwx /tmp/.docker
+  - type: "host_script"
+    title: "transfer docker config to remote host"
+    script: |-
+      scp -F ./.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.docker/config.json openshiftdevel:/tmp/.docker/
+  - type: "script"
+    title: "push origin-metrics release"
+    repository: "origin-metrics"
+    script: |-
+      sudo chmod a+rw /tmp/.docker/config.json
```
Comment (smarterclayton) [inline sjb/config/test_cases/push_origin_metrics_release.yml]: Please keep the tagging login from the logging job.  If push release doesn't support multi tagging, please add that to your hack/push-release.

Comment (smarterclayton) [reply sjb/config/test_cases/push_origin_metrics_release.yml]: Oops, ignore this.  I haven't added it yet.  As long as this consistent with logging_release you're good for now.

**Comment by jcantrill** [comment]
@stevekuznetsov  if this meets your lofty expectations can you please approve and merge..

**Comment by jcantrill** [comment]
@stevekuznetsov @jupierce  Is this withing your realm to approve and merge?

**Comment by jupierce** [comment]
@jcantrill will need @stevekuznetsov eyes

**Comment by stevekuznetsov** [comment]
Eek sorry, missed this

**Pull request merged**